app/components/post_textbox/post_textbox.js

```diff
@@ -260,44 +264,46 @@ export default class PostTextbox extends PureComponent {
                     >
-                        <Icon
-                            size={30}
-                            color={changeOpacity('#fff', 0.9)}
-                            name='md-add'
-                        />
-                    </TouchableHighlight>
-                    <View style={style.inputContainer}>
-                        <TextInputWithLocalizedPlaceholder
-                            ref='input'
-                            value={this.props.value}
-                            onChangeText={this.handleTextChange}
-                            onSelectionChange={this.handleSelectionChange}
-                            onContentSizeChange={this.handleContentSizeChange}
-                            placeholder={placeholder}
-                            placeholderTextColor={changeOpacity('#000', 0.5)}
-                            onSubmitEditing={this.sendMessage}
-                            multiline={true}
-                            underlineColorAndroid='transparent'
-                            style={[style.input, {height: textInputHeight}]}
-                        />
-                        <TouchableHighlight
-                            onPress={this.sendMessage}
+                        <TouchableOpacity
+                            onPress={this.showFileAttachmentOptions}
                             style={style.buttonContainer}
                         >
                             <Icon
-                                name='ios-arrow-round-up'
-                                size={34}
-                                color={theme.linkColor}
+                                size={30}
                                 style={{marginTop: 2}}
+                                color={changeOpacity(theme.centerChannelColor, 0.9)}
+                                name='md-add'
                             />
-                        </TouchableHighlight>
+                        </TouchableOpacity>
+                        <View style={style.inputContainer}>
+                            <TextInputWithLocalizedPlaceholder
+                                ref='input'
+                                value={this.props.value}
+                                onChangeText={this.handleTextChange}
+                                onSelectionChange={this.handleSelectionChange}
+                                onContentSizeChange={this.handleContentSizeChange}
+                                placeholder={placeholder}
+                                placeholderTextColor={changeOpacity('#000', 0.5)}
+                                onSubmitEditing={this.sendMessage}
+                                multiline={true}
+                                underlineColorAndroid='transparent'
+                                style={[style.input, {height: Math.min(this.state.contentHeight, MAX_CONTENT_HEIGHT)}]}
+                            />
+                            <TouchableOpacity
```

Comment:
I think the ticket says something about only showing the send button when the user has typed something or if it has attached files